src/language-features/casting.md

```diff
@@ -0,0 +1,28 @@
+---
+minutes: 3
+---
+
+# Casting
+
+Rust has no _implicit_ type conversions, but does support explicit casts with
+`as`. These generally follow C semantics where those are defined.
```

Comment:
`as` conversions are a pretty common source of bugs and are discouraged where possible. It would be good if we could introduce these at the same time as the Into/From/TryInto traits so we can discuss best practices. At the very least we should warn that `as` can change the meaning of values (`0xffu8 as i8`) and lose data (`256u16 as u8`).